mne/decoding/tests/test_base.py

```diff
@@ -12,6 +13,95 @@
 from mne.decoding import Scaler
 
 
+def _make_data(n_samples=1000, n_features=5, n_targets=3):
+    """Generate some testing data.
+
+    Parameters
+    ----------
+    n_samples : int
+        The number of samples.
+    n_features : int
+        The number of features.
+    n_targets : int
+        The number of targets.
+
+    Returns
+    -------
+    X : ndarray, shape (n_samples, n_features)
+        The measured data.
+    Y : ndarray, shape (n_samples, n_targets)
+        The latent variables generating the data.
+    A : ndarray, shape (n_features, n_targets)
+        The forward model, mapping the latent variables (=Y) to the measured
+        data (=X).
+    """
+
+    # Define Y latent factors
+    cov_Y = np.eye(n_targets) * 10 + np.random.rand(n_targets, n_targets)
+    mean_Y = np.random.rand(n_targets)
+    Y = np.random.multivariate_normal(mean_Y, cov_Y, size=n_samples)
+
+    # Put a scale and offset to Y
+    Y += np.random.rand(n_targets)  # Put an offset
+
+    # The Forward model
+    A = np.random.randn(n_features, n_targets)
+
+    X = Y.dot(A.T)
+    X += np.random.randn(n_samples, n_features)  # add noise
+    X += np.random.rand(n_features)  # Put an offset
+
+    return X, Y, A
+
+
+def __make_data(n_samples=1000, n_features=5, n_targets=3, noise_scale=2):
```

Comment:
Yay, I was hoping my Gist code would end up as a unit test :).
But do we really need two functions that are called `make_data` and generate test data?